include/crow/http_request.h

```diff
@@ -31,6 +32,15 @@ namespace crow // NOTE: Already documented in "crow/app.h"
         return empty;
     }
 
+    struct bad_request : public std::runtime_error
+    {
+        bad_request(const std::string& what_arg)
+            : std::runtime_error(what_arg) {}
+
+        bad_request(const char* what_arg)
+            : std::runtime_error(what_arg) {}
+    };
+
```

Comment:
apologies, I just noticed this, wouldn't it be best to have this be either in `utility.h` or another header file specifically for exceptions?

Reply:
well i did not see any exception specific header. i am fine with either utility.h or an "exceptions.h` or whatever name you wish :)

Reply:
`exceptions.h` sounds good!